pycbc/workflow/tmpltbank.py

```diff
@@ -288,6 +288,28 @@ def setup_tmpltbank_without_frames(workflow, output_dir,
         
     return tmplt_banks
 
+def _bank_name_to_file(cp, ifos, global_seg, key, tags):
+    user_tag     = "PREGEN_TMPLTBANK"
+    pre_gen_bank = cp.get_opt_tags('workflow-tmpltbank', key, tags)
+    lfn          = urlparse.urlparse(pre_gen_bank).path
+
+    # If it's a file
+    if lfn == pre_gen_bank:
```

Comment:
Shouldn't the following be in the File class itself? If we always need to use a nonlocal site for anything but a standard path or file url, it should be the default. Can you make a method of the class that is "add_pfn", or something like that.